src/routes/getting-started/table.svelte

```diff
@@ -158,6 +158,24 @@
 
 		'mac,Python,X64,DefaultCPU': 'pip install onnxruntime',
 
+		'mac,Python,X64,CoreML': 'pip install onnxruntime',
+
+		'mac,Python,ARM64,CoreML': 'pip install onnxruntime',
+
+		'mac,JS,ARM64,CoreML': 'npm install onnxruntime-react-native',
```

Comment:
should it be onnxruntime-node?
https://github.com/microsoft/onnxruntime/blob/73356fc1920edd8df13ed62ac169de5a78923ee8/src/routes/getting-started/table.svelte#L313

does the JS package have the CoreML EP enabled? I'm not familiar with it.

Reply:
https://github.com/microsoft/onnxruntime/blob/df740d7d152dfd04d29f4310a02d41b68f010b37/js/node/src/session_options_helper.cc#L80-L84

from here, looks like CoreML EP is enabled.

Reply:
confirmed that the support with the combination here about "JS, CoreML" is not there yet. so maybe we can remove this line here for now.
```
'mac,JS,ARM64,CoreML': 'npm install onnxruntime-react-native',
```